src/runtime_src/core/pcie/tools/awssak/main.cpp

```diff
@@ -20,5 +20,13 @@
 
 int main(int argc, char *argv[])
 {
-    return xcldev::xclAwssak(argc, argv);
+    int ret = 0;
+
+    try {
+      ret = xcldev::xclAwssak(argc, argv);
+    } catch (const std::exception& ex) {
+      std::cout << ex.what() << std::endl;
+    }
+
+    return ret;
 }
```

Comment:
As written above, the function will return 0 when an exception is thrown.  Please do:
```
try {
  return xcldev::xclAwssak(...);
}
catch (const std::exception& ex) {
   ...;
}
catch (...) {
  .....;
}
return 1;
```